File: 2022/Tachi/app/src/main/java/com/creator/tachi/util/ProgressUtil.java
package com.creator.tachi.util;

import android.app.Dialog;
import android.content.Context;
import android.view.Window;

import com.creator.tachi.R;

/**
 * Created by skcrackers on 5/27/16.
 */
public class ProgressUtil extends Dialog {
    public ProgressUtil(Context context) {
        super(context);
        requestWindowFeature(Window.FEATURE_NO_TITLE);
        setContentView(R.layout.view_progress_util);
    }
}

File: 2025/LiveMidmeal/app/src/main/java/com/creator/livemidmeal/util/ProgressUtil.kt
package com.creator.livemidmeal.util

import android.app.Dialog
import android.content.Context
import android.view.Window
import com.creator.livemidmeal.R

/**
 * Created by skcrackers on 5/27/16.
 */
class ProgressUtil(context: Context?) : Dialog(context!!) {
    init {
        requestWindowFeature(Window.FEATURE_NO_TITLE)
        setContentView(R.layout.view_progress_util)
    }
}